CRM/Upgrade/Incremental/sql/5.11.alpha1.mysql.tpl

```diff
@@ -1 +1,2 @@
 {* file to handle db changes in 5.11.alpha1 during upgrade *}
+ALTER TABLE civicrm_action_log CHANGE COLUMN reference_date reference_date datetime;
```

Comment:
@Dawnthorn what is the reasoning behind this?

Reply:
It's so that this methodology of storing the reference date will work with all the kinds of entities, not just memberships. For example, for activities, the reference date is activity_date_time which is a datetime field. Making this field date time makes it work for both dates and datetimes.